integration-test/src/test/java/org/bitcoinj/wallet/WalletAccountPathTest.java

```diff
@@ -64,6 +67,10 @@ void walletStructurePathTest2(KeyChainGroupStructure structure, HDPath expectedP
 
         // Then the account path is as expected
         assertEquals(expectedPath, wallet.getActiveKeyChain().getAccountPath());
+
```

Comment:
Would it make sense to PR and merge this change right away?

Reply:
The change asserting that the first receive address is not null?